gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -861,13 +860,14 @@ func GetSecretForPV(restSecretNamespace, restSecretName, volumePluginName string
 }
 
 var (
-	master     = flag.String("master", "", "Master URL")
-	kubeconfig = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
-	id         = flag.String("id", "", "Unique provisioner identity")
+	master          = flag.String("master", "", "Master URL")
+	kubeconfig      = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
+	id              = flag.String("id", "", "Unique provisioner identity")
+	provisionerName = "gluster.org/glusterblock"
 )
 
 func main() {
-	klog.InitFlags(nil)
+	//klog.InitFlags(nil)
```

Comment:
this looks suspect. Was it crashing until you removed this line?

Reply:
Yes.  The app would crash out on startup.

I'm not experienced with GO, but did some digging on the generated error and the app behavior.  What I found seemed to indicate that klog.InitFlags, or something similar was already being called higher up in the bootstrap process for the application.  So, I removed it.  Looking back through the code, it looked like klog had recently been added to the app as a replacement for glog.

Reply:
OK, thanks, you're right to remove it. glog->klog migration happened in this git repo and I was not paying attention, unfortunate that it broke.